docker/service.go

```diff
@@ -273,3 +279,13 @@ func (s *Service) Containers() ([]project.Container, error) {
 
 	return result, nil
 }
+
+func (s *Service) SpecificiesHostPort() bool {
+	for _, port := range s.serviceConfig.Ports {
+		if strings.ContainsRune(port, ':') {
+			return true
+		}
+	}
```

Comment:
Thanks for catching this! It looks like <IP>::<port> is the only case that fails, so it should be sufficient to check that "::" also isn't in the port string, right?